spec/ruby/core/module/refine_spec.rb

```diff
@@ -762,6 +787,120 @@ def foo
 
       result.should == "foo"
     end
+
+    it "looks only in the refined class even if there is another active refinement" do
+      refined_class = ModuleSpecs.build_refined_class
+
+      refinement = Module.new do
+        refine refined_class do
+          def bar
+            "foo from refinement"
+          end
+        end
+      end
+
+      refinement_with_super = Module.new do
+        refine refined_class do
+          def bar
+            super
+          end
+        end
+      end
+
+      -> {
+        Module.new do
+          using refinement
+          using refinement_with_super
+          refined_class.new.bar
+        end
+      }.should raise_error(NoMethodError)
+    end
+
+    it "looks in the another active refinement if super called from included modules
+        (https://bugs.ruby-lang.org/issues/16977)" do
```

Comment:
Could you move that link as a comment above the `it`?
Multi-line or too long spec descriptions should be avoided in general.